x-pack/plugins/gis/server/routes.js

```diff
@@ -17,32 +17,31 @@ export function initRoutes(server) {
   const serverConfig = server.config();
   const mapConfig = serverConfig.get('map');
 
-  const emsV2 = new EMS_V2({
+  const emsClient = new EMSClientV66({
     kbnVersion: serverConfig.get('pkg.version'),
-    license: server.plugins.xpack_main.info.license.getUid(),
     manifestServiceUrl: mapConfig.manifestServiceUrl,
-    emsLandingPageUrl: mapConfig.emsLandingPageUrl
+    landingPageUrl: mapConfig.emsLandingPageUrl
   });
 
   server.route({
     method: 'GET',
     path: `${ROOT}/data/ems`,
     handler: async (request) => {
 
-      if (!request.query.name) {
+      if (!request.query.id) {
+        console.warn('Must supply id parameters to retrieve EMS file');
```

Comment:
Server messages should be logged with server.log, `server.log(['warning', message);`